test/cargo_build_script/BUILD.bazel

```diff
@@ -27,3 +27,35 @@ rust_test(
     edition = "2018",
     deps = [":tools_exec_build_rs"],
 )
+
+environment_group(
+    name = "foo_environment",
+    defaults = [":foo"],
+    environments = [
+        ":foo",
+        ":bar",
+    ],
+)
+
+environment(
+    name = "foo",
+)
+
+environment(
+    name = "bar",
+)
+
+cargo_build_script(
+    name = "empty_build_script",
+    srcs = ["do_nothing.rs"],
+    compatible_with = [":bar"],
+    edition = "2018",
+)
+
+rust_test(
```

Comment:
I'm not sure the test adds much value as it's a thing you need to manually run to observe, but 👍